developer-docs-site/docs/nodes/ait/node-requirements.md

```diff
@@ -67,6 +67,20 @@ We recommend nodes have at least 300GB of disk space to ensure adequate storage
 
 Bandwidth requirement: 1 Gbps
 
+### Ports
+
+When you are running a validator node, you are required to open network ports on your node to allow other nodes to connect to you. For fullnodes this is optional.
+
+There are three types of Aptos networks. Your node can be configured so that each of these networks can connect to your node using a different port on your node. 
+
+1. The validator network: A validator node connects to this network.
+2. The public network. A public fullnode connects to this network. 
+3. The validator fullnode network (VFN): A validator fullnode connects to this network. The VFN allows the validator fullnode to connect to the specific validator). 
```

Comment:
1. VFN generally means "Validator fullnode". 😄 If you want to refer to the network, use "VFN network".
2. "The VFN network allows the validator fullnode to connect to the specific validator as well as other VFNs".